Proposals/TypeNetwork_ParametricAxes/ProposalSummary_ytde.md

```diff
@@ -48,6 +48,8 @@ collisions in this area suffered by fonts without it) but not an area they may k
 This name fits into the systematic structure of the overall system of proposed axes: Y
 dimension transparency of descenders.
 
+ytde is measured from the baseline to the bottom of descender
```

Comment:
Since this is part of the portion that would be published in the registry, can you please formulate as a full, punctuated sentence; e.g.,

"The ytde value is measured from the baseline to the bottom of the descender.

Also, wouldn't this make sense to be added to the Scale Interpretation field, rather than Additional Information?